helpers/settings.go

```diff
@@ -138,7 +141,41 @@ func (s *Settings) InitSettings(envVars *EnvVars, env *cfenv.App) (retErr error)
 		return GenerateRandomString(32)
 	}
 
+	// Initialize CSRF key
+	var err error
+	// First, try the new env variable
+	s.CSRFKey, err = hex.DecodeString(envVars.String(CSRFKeyEnvVar, ""))
+	if err != nil {
+		return err
+	}
+
+	// Fall back to legacy key variable and format - consider printing deprecation warning
+	if len(s.CSRFKey) == 0 {
+		s.CSRFKey = []byte(envVars.String(LegacySessionKeyEnvVar, ""))
```

Comment:
print something out saying falling back to `SESSION_KEY` for `CSRF_KEY`

Reply:
Fixed.